src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,24 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int bulldozerYear;
```

Comment:
Maybe you mean age?

Reply:
No, I mean the year when bulldozer was made

Reply:
I'm sorry, I can't say that I fully understood - you mean a enum object or  just an integer variable with different name for each machine type (or I understood you wrong?)